{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "cd7ab9a5_c348254d",
        "filename": "roles/defaults/vars/main.yml",
        "patchSetId": 2
      },
      "lineNbr": 154,
      "author": {
        "id": 1000003
      },
      "writtenOn": "2025-01-19T02:06:25Z",
      "side": 1,
      "message": "@mnaser@vexxhost.com the neutron_ovn_vpn image didn\u0027t exist in stable/2023.2 branch, if we need add it?",
      "revId": "303f61df4e744bf47e34adf8c634bf702f33223f",
      "serverId": "a369f105-6bff-480f-8aaf-eafd1f85bb44"
    }
  ]
}